dbt_project.yml

```diff
@@ -10,7 +10,7 @@ models:
       +materialized: ephemeral
 vars:
   asana:
-    project_task: "{{ ref('stg_asana__project_task') }}"
+    project_task: "{{ ref('stg_asana__project_task_tmp') }}"
```

Comment:
I'm not sure if this is the best way to handle this, but I wanted access to the `_fivetran_synced` column to help set the "primary" project in a repeatable way.  I'm open to other suggestions!